cql2pgjson/src/test/java/org/folio/cql2pgjson/CompoundIndexTest.java

```diff
@@ -41,4 +41,36 @@ public void compoundIndexSQLExpression2() throws Exception {
     String expected = "WHERE to_tsvector('simple', lower(concat_space_sql(jsonb->>'field1', jsonb->>'field2'))) @@ replace((to_tsquery('simple', ('''Boston''')) && to_tsquery('simple', ('''MA''')))::text, '&', '<->')::tsquery";
     assertEquals(expected, sql);
   }
+  @Test
+  public void compoundIndexMultiFieldnames3() throws Exception {
+    CQL2PgJSON cql2pgJson = new CQL2PgJSON("tablec");
+    cql2pgJson.setDbSchemaPath("templates/db_scripts/compoundIndex.json");
+    String sql = cql2pgJson.toSql("tablecftindex = Boston MA").toString();
+    String expected = "WHERE to_tsvector('simple', concat_space_sql(tablec.jsonb->>'firstName' , tablec.jsonb->>'lastName')) @@ replace((to_tsquery('simple', ('''Boston''')) && to_tsquery('simple', ('''MA''')))::text, '&', '<->')::tsquery";
+    assertEquals(expected, sql);
+  }
+  @Test
+  public void compoundIndexMultiFieldnames4() throws Exception {
+    CQL2PgJSON cql2pgJson = new CQL2PgJSON("tablec");
+    cql2pgJson.setDbSchemaPath("templates/db_scripts/compoundIndex.json");
+    String sql = cql2pgJson.toSql("tablecginindex == Boston MA").toString();
+    String expected = "WHERE lower(concat_space_sql(tablec.jsonb->>'firstName' , tablec.jsonb->>'lastName')) LIKE lower('Boston MA')";
+    assertEquals(expected, sql);
+  }
+  @Test
+  public void compoundIndexMultiFieldnames5() throws Exception {
+    CQL2PgJSON cql2pgJson = new CQL2PgJSON("tabled");
+    cql2pgJson.setDbSchemaPath("templates/db_scripts/compoundIndex.json");
+    String sql = cql2pgJson.toSql("tabledftindex = Boston MA").toString();
+    String expected = "WHERE to_tsvector('simple', concat_space_sql(tabled.jsonb->'proxy'->'personal'->>'city' , tabled.jsonb->'proxy'->'personal'->>'state')) @@ replace((to_tsquery('simple', ('''Boston''')) && to_tsquery('simple', ('''MA''')))::text, '&', '<->')::tsquery";
+    assertEquals(expected, sql);
+  }
+  @Test
+  public void compoundIndexMultiFieldnames6() throws Exception {
+    CQL2PgJSON cql2pgJson = new CQL2PgJSON("tabled");
+    cql2pgJson.setDbSchemaPath("templates/db_scripts/compoundIndex.json");
+    String sql = cql2pgJson.toSql("tabledginindex == Boston MA").toString();
+    String expected = "WHERE lower(concat_space_sql(tabled.jsonb->'proxy'->'personal'->>'city' , tabled.jsonb->'proxy'->'personal'->>'state')) LIKE lower('Boston MA')";
+    assertEquals(expected, sql);
+  }
```

Comment:
Can you rename the test methods?
The prefix "compoundIndex" is redundant and should be removed because it is in the CompoundIndexTest class.
The number should be avoided and replaced by "gin", "ft", "subfield", etc.